src/main/resources/schemas/schemaFileSchema.json

```diff
@@ -140,6 +160,10 @@
                       {
                         "type": "string",
                         "pattern": "^extra"
+                      },
+                      {
+                        "type": "string",
+                        "pattern": "^literal"
```

Comment:
We could just merge this with the above, `^(extra|literal)`. Up to you, no change required here.